src/app.py

```diff
@@ -303,20 +303,26 @@
         )
     # Perform Tiled SD upscale (EXPERIMENTAL)
     elif args.sdupscale:
-        config.lcm_diffusion_setting.strength = 0.3 if args.use_openvino else 0.1
+        if args.use_openvino:
+            config.lcm_diffusion_setting.strength = 0.3
```

Comment:
We need to use a low value here (0.1 is best) but 0.1 causes problem with OpenVINO so we can keep as it is.